### ISSUE TITLE
Unit tests don't work on node@8.x

### ISSUE THREAD

**Issue by iTOYS** [Microsoft/vscode-react-native#467]
## Actual Behavior
1. Tests failed

## Expected Behavior
1. Tests passed

## Software versions

- NodeJS version: 8.1.3

## Outputs (Include if relevant)
- Output of the Debug Console (View -> Toggle Debug Console):
```
Error: Unsupported Node version: 8.1.3
    at Object.<anonymous> (/Users/TOYS/projects/vscode-react-native/node_modules/mock-fs/lib/index.js:38:9)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/TOYS/projects/vscode-react-native/src/test/debugger/android/androidPlatform.test.ts:6:1)
```

**Comment by aschrijver** [comment]
With Node `8.1.3` and VS Code `1.13.1` I didn't get the 'Unsupported...' error, but breakpoints where just not hit. This may be caused with issues of `inspector` protocol in v8. See [this stackoverflow question](https://stackoverflow.com/questions/45056952/debugging-jest-unit-tests-with-breakpoints-in-vs-code-with-react-native/45073796) for details.

I downgraded node to `7.10.1` but now the issue is that code is run directly from disk, rather than through the generated js bundle, leading to my browserify shims to be ignored.
My tests succeed where they should fail. See: https://github.com/Microsoft/vscode/issues/26782

**Comment by iTOYS** [comment]
@aschrijver 
This issue about extension's self-tests.

**Comment by vladimir-kotikov** [comment]
@aschrijver this PR is not about running/debugging tests for react-native applications. This is just about this extension's tests. If you're having any troubles with that, please file a separate issue describing your problem.

As for loading code from disk rather from packager - this is a thing we can't avoid as we need to process bundle before loading it into Node. However I don't see any negative consequenses of that but if you have any issues - open a new issue and explain your use case so we could try to investigate.

Thanks

**Comment by vladimir-kotikov** [comment]
@iTOYS Dupe %)

**Comment by aschrijver** [comment]
Thanks @iTOYS @vladimir-kotikov for pointing this out!

**Comment by vladimir-kotikov** [comment]
Resolved in #506
